### ISSUE TITLE
Query Results Window Not Formatting Fields that Contain Multiple Lines

### ISSUE THREAD

**Issue by cjones722** [serge-rider/dbeaver#2192]
The Query Results Window in DBeaver is not formatting the fields that contains multiple lines.  A **¶** character is displayed instead of having the next part of the text displayed on a different line.

Have tried going to _"Preferences > Database > Result Sets > Binaries"_ in DBeaver and changing the "String encoding" from the default UTF-8 value.

The Query Results Window in DBeaver displays the **¶** character instead of going to the next line of text regardless of the DB connection type (PostgreSQL or Oracle).

**Example:**

- DBeaver Version 4.2.1
- PostgreSQL Version:  9.4.7

Here the text for the log_message field is displayed on one line with ¶ where it should go to the next line.
```

log_message                                                                                                                                                                                          
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Sep 10, 2017 1:57:36 AM - User Secondary Institution Role [create/update, 1......, STUDENT]¶Institution Role'STUDENT' applied to '1......' successfully.¶1......|STUDENT ¶Field mapping successful.¶ 

```

This is how the log_message field properly is displayed when connecting to the Postgres DB via the psql client and querying the respective table:
```
                                         log_message
---------------------------------------------------------------------------------------------
 Sep 10, 2017 1:57:36 AM - User Secondary Institution Role [create/update, 1......, STUDENT]+
 Institution Role'STUDENT' applied to '1......' successfully.                               +
 1......|STUDENT                                                                            +
 Field mapping successful.                                                                  +

```

**Comment by serge-rider** [comment]
Yes, DBeaver converts all string values into single line representation.

However if you copy the value it will be copied with appropriate line feed. 
If you need to see the value in source formatting you can open "Value viewer" panel (in result bottom toolbar).
Having line feeds right in grid representation usually isn't wanted (and also complicated in implementation). I can mark this a feature request if you think that this is a really helpful feature.

**Comment by cjones722** [comment]
Thank you for the feedback.  Yes, please mark this as a feature request.

**Comment by aderchox** [comment]
One thing that is bothering me (if I'm not mistaken and it really relates to this issue) is that when I copy the one-line representation of a multiline value, I cannot paste it back to the same field afterwards! So it's a multiline text, I see it in one line in dbeaver, then I copy it, but I won't be able to paste it back to the same field or another field which accepts text.

**Comment by bitloi** [comment]
@E1izabeth Sorry to bother you again, would you please let me know if I can work on this issue?
